README.md

```diff
@@ -6,7 +6,7 @@
 
 > **Atention**
 >
-> This module and [Module 2](https://github.com/Arquisoft/Agents_i3a) had been added to heroku Continuos Delivery, so you can test the application at https://mana-module.herokuapp.com for web services and https://mana-module.herokuapp.com/sensor-feed for REST services.
+> This module and [Module 2](https://github.com/Arquisoft/Agents_i3a) had been added to heroku Continuos Delivery, so you can test the application at http://i3a-3.herokuapp.com for web services and http://i3a-3.herokuapp.com/sensor-feed for REST services.
```

Comment:
Please notice it is https and not http, this may sound like unimportant but new versions of safari block any http connection allowing only https. Besides heresy already have a wild child ssl certificate for every app installed.